attest/attest.go

```diff
@@ -201,6 +203,10 @@ type AttestationParameters struct {
 	// the TPM Part 2 Structures specification, available at
 	// https://trustedcomputinggroup.org/wp-content/uploads/TPM-Main-Part-2-TPM-Structures_v1.2_rev116_01032011.pdf
 
+	// CertifyingKey is a public key used for the certification of Public. Only
+	// TPM 2.0 is supported and the key is encoded as a TPMT_PUBLIC structure.
+	CertifyingKey []byte
```

Comment:
AttestationParameters are generated by the client and attacker controlled. I think we should just drop this field as isn't never verified and basically should never be used. Make this an argument of VerifyOpts?

Reply:
Certifying key is a parameter of the key certification process, so it felt natural to include it within AttestationParameters. Yes, the verifier needs to make sure that CertifyingKey is trusted.